server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleTaskService.java

```diff
@@ -62,14 +67,29 @@ public class ScheduleTaskService {
     @Autowired
     private ScheduleTaskRepository scheduleTaskRepository;
 
+    @Autowired
+    private ScheduleTaskShardRepository scheduleTaskShardRepository;
+
     @Autowired
     private QuartzJobService quartzJobService;
 
+    private final ScheduleTaskMapper scheduleTaskMapper = ScheduleTaskMapper.INSTANCE;
+
     @Value("${odc.log.directory:./log}")
     private String logDirectory;
 
     private static final String LOG_PATH_PATTERN = "%s/scheduleTask/%s-%s/%s/log.%s";
 
+    public ScheduleTaskResp detail(Long id) {
+        ScheduleTaskEntity entity = nullSafeGetById(id);
```

Comment:
`ScheduleService#nullSafeGetByIdWithCheckPermission` returns the `ScheduleEntity`

`ScheduleService#detailScheduleTask` call the method `ScheduleService#nullSafeGetByIdWithCheckPermission` and ignore the return value and call `nullSafeGetById`, why should we query schedule entity twice?